index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+</head>
+<body>
```

Comment:
- [ ] Good job working on the mobile-first 👍. but for best practice, you should use this form to complete the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=The%202%20sections%20you%20need%20to%20build%20for%20this%20project%20are%3A) 😃. 

````
<body>
        <header>
                 ... the toolbar (or header)
        </header>

       <main>
               <section>
                      ... the headline section (right after the header)
               </section>

               <section>
               </section>
       </main>
</body>

````

in the toolbar (or header)
![ttt](https://user-images.githubusercontent.com/57408419/227212069-8578a3c8-38d5-45e3-ae84-7324311aa1f9.JPG)

````
<header>
            <a>logo</a>
            <nav>
                   icon ham bagger
            </nav>
</header>

````

- [ ] Also make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [requied](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture0](https://user-images.githubusercontent.com/57408419/227212765-74ecfd0c-fa24-4ff7-9fff-b3bc42fb1a88.JPG)| |![Capture](https://user-images.githubusercontent.com/57408419/227213630-6f2fed06-c1b7-42d2-a200-debfec0f3dd7.JPG)| |